client/src/components/settings/certification.tsx

```diff
@@ -206,11 +206,11 @@ function CertificationSettings(props: CertificationSettingsProps) {
   };
 
   type CertName = keyof ProjectMap | keyof LegacyProjectMap;
-  function renderCertifications(certName: CertName) {
-    const { t } = props;
+  const Certification = ({ certName }: { certName: CertName }) => {
+    const { t } = useTranslation();
```

Comment:
I'd either use `useTranslation` everywhere (and get rid of `withTranslation`) or pass the `t` function to this. Either is fine, but mixing them is just confusing.